book/appendix/virtualenv.md

```diff
@@ -0,0 +1,67 @@
+# Walaroo and Virtualenv
+
+We recommend using [virtualenv](https://virtualenv.pypa.io/en/stable/) to handle your application's dependencies.
+
+While not required to run Wallaroo applications, virtualenv will ensure that your application and development environment are isolated from the rest of your Python installation(s) on your computer. In addition, virtualenv will ensure that machida uses the correct version of Python if you have multiple versions installed on your computer.
+
+## Installing Virtualenv
+
+Follow the instructions on [the virtualenv website](https://virtualenv.pypa.io/en/stable/installation/).
+
+## Setting up Virtualenv for your Application
+
+To create a virtualenv for your application, run the following command
+
+```bash
+virtualenv --python=python2 ENV
+```
+
+This will create a directory named `ENV` and set up all of the scripts and dependencies required to run an isolated python virtual environment.
+
+OS X users should not omit the `--python` argument to `virtualenv` in order to work around a known problem with Python 2.7.
```

Comment:
I think this could be a little confusing. We've already told people that they need to use `--python`. Saying "OS X users should not omit ..." makes it seem like there's a reason they might have omitted it.

I'd either omit this or reword it to something like "OS X users MUST use the `--python` argument to `virtualenv` in order to work around a known problem with Python 2.7, it may not be necessary for users of other operating systems."

Reply:
@aturley looks like `--python=python2` works on OSX, so I'm just going to omit this.